src/snowflake/snowpark/column.py

```diff
@@ -613,10 +613,10 @@ def bitxor(self, other: Union[ColumnOrLiteral, Expression]) -> "Column":
         build_expr_from_snowpark_column_or_python_val(ast.rhs, other)
         return Column(BitwiseXor(Column._to_expr(other), self._expression), ast=expr)
 
-    def __neg__(self) -> "Column":
+    def __neg__(self, _caller_frame_depth=2) -> "Column":
```

Comment:
Is there ever a possibility of passing a different value here? If not (and I don't think there is), you should just pass the constant 2 in line 619. Same everywhere except where you actually have a use case for parametrization.

Reply:
Oh, I see you're actually using this – great!